src/server/protocol.ts

```diff
@@ -2436,6 +2436,27 @@ namespace ts.server.protocol {
         openFiles: string[];
     }
 
+    export type LargeFileReferencedEventName = "largeFileReferenced";

```

Comment:
Why is this code duplicated? I guess that’s one public and the other private, but why aren’t they shared?

Reply:
That's because we want to be able to build protocol.ts separately as a standalone to protocol.d.ts which editors consume.